c7n/resources/cloudfront.py

```diff
@@ -436,3 +438,109 @@ def process_distribution(self, client, distribution):
                 "Exception trying to force ssl on Distribution: %s error: %s",
                 distribution['ARN'], e)
             return
+
+
+@Distribution.action_registry.register('update-distribution')
```

Comment:
one last change, i'd like to have this renamed to set-attributes with the update key renamed to attributes since we're trying more to have policies express idempotent intent.